### PR TITLE
Add some info about initial charge.

### PR DESCRIPTION
**Pull request by skasti**
Add some info about how you need initial charge to require payment for agreement activation, and improve the description of what an agreement is.

### CODE REVIEW

**Comment by cloveras** [comment]
Haha, I created a different PR without knowing about this one. See if you want anything from this one, and if not I'll just close it. https://github.com/vippsas/vipps-recurring-api/pull/262

File: vipps-recurring-api.md [hunk 1]
```diff
@@ -392,6 +390,8 @@ See
 [Vipps landing page](https://github.com/vippsas/vipps-developers/blob/master/common-topics/vipps-landing-page.md)
 from Common topics, for more details about the landing page.
 
+**Note:** If payment should be required to activate an agreement, you need to specify an initial charge. If you are dealing with physical goods, this should be a RESERVE_CAPTURE, but for digital goods where the customer instantly gains access, DIRECT_CAPTURE might be easier to manage. See [Initial charge](#initial-charge)
```
Comment (cloveras) [inline vipps-recurring-api.md:393]: In other APIs we try to get away from "direct capture", as it's impossible to cancel then. See: https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/reserve-and-capture

**Review by cloveras — APPROVED** [review]
Looks good! See comment, and my slightly overlapping PR.

**Review by ameliemaucade — APPROVED** [review]

File: vipps-recurring-api.md [hunk 2]
```diff
@@ -312,9 +312,7 @@ for each order, or some similar, unique and readable pattern.
 
 ## Agreements
 
-An agreement is between the Vipps user and the merchant.
-Think of it as a subscription.
-An agreement has payments, called [charges](#charges).
+An agreement is between the Vipps user and the merchant. Think of it as a payment agreement that allows you (the merchant) to recurringly charge the customer without them having to manually approve every time. See [charges](#charges).
```
Comment (rebekaburnett) [inline vipps-recurring-api.md:315]:
```suggestion
An agreement is between the Vipps user and the merchant. Think of it as a payment agreement that allows you (the merchant) to charge the customer at the agreed time interval without requiring them to manually approve every time. See [charges](#charges).
```

Comment (rebekaburnett) [reply vipps-recurring-api.md:315]: Just a little word-smithing - I think this is how we would say it. :-)

**Review by rebekaburnett — APPROVED** [review]

File: vipps-recurring-api.md [hunk 3]
```diff
@@ -1637,4 +1637,4 @@ Sign up for our [Technical newsletter for developers](https://vippsas.github.io/
 [refund-charge-endpoint]: https://vippsas.github.io/vipps-developer-docs/api/recurring#tag/Charge-v2-endpoints/operation/RefundCharge
 [userinfo-endpoint]: https://vippsas.github.io/vipps-developer-docs/api/recurring#tag/Userinfo-Endpoint/operation/getUserinfo
 [access-token-endpoint]: https://vippsas.github.io/vipps-developer-docs/api/recurring#tag/Authorization-Service/operation/getAccessToken
-[vipps-test-environment]: https://github.com/vippsas/vipps-developers/blob/master/developer-resources/test-environment.md
\ No newline at end of file
+[vipps-test-environment]: https://github.com/vippsas/vipps-developers/blob/master/developer-resources/test-environment.md
```
Comment (rebekaburnett) [inline vipps-recurring-api.md:1640]:
```suggestion
[vipps-test-environment]: https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/developer-resources/test-environment
```

**Pull request merged**